ansible/roles/software/datagov-deploy-fgdc2iso/templates/fgdc2iso.nginx.conf.j2

```diff
@@ -0,0 +1,19 @@
+server {
```

Comment:
Let's use the [geerlingguy.nginx role](https://github.com/geerlingguy/ansible-role-nginx) instead. This will make it easier to share common nginx settings with dashboard and wordpress in the future. Currently dashboard and wordpress use geerlingguy.nginx for the package install only, but should use it for nginx configuration as well (obviously out of scope for this story).

There's a handful of missing things I see in this configuration:
- Prevent [header injection](https://github.com/GSA/datagov-deploy-wordpress/blob/f701837c70e3ff968470d5187521d31b5ef6fcd7/templates/datagov.nginx.conf#L9) in the redirect (not sure if nginx has since come up with a better, modern solution)
- Disable [TLS < 1.2](https://github.com/GSA/datagov-deploy-wordpress/blob/f701837c70e3ff968470d5187521d31b5ef6fcd7/templates/datagov.nginx.conf#L35)
- Disable weak ciphers

Reply:
Cool. Did not know the role can do that.